style/comp-rest.css

```diff
@@ -43,4 +43,11 @@ main{
     width: 100%;
     border: none;
     min-height: 50vh;
+}
+.component-button{
+    font-size: 1.5rem;
+    margin: 1rem;
+}
+.right-button{
+    float: right;
```

Comment:
You can add a new line at the end of each file

Reply:
ok I will consider